package.json

```diff
@@ -96,6 +96,11 @@
     "react-native-localize": "^1.4.0",
     "react-native-modal": "^11.5.6",
     "react-native-modalbox": "^1.7.1",
+    "react-native-open-maps": "^0.3.4",
+    "react-native-orientation-locker": "^1.1.6",
+    "react-native-paper": "^3.2.1",
+    "react-native-prompt-android": "^1.1.0",
+    "react-native-reanimated": "^1.3.0",
```

Comment:
All the packages added here already exists in that file in a more recent version except react-native-prompt-android. You are duplicating them. npm should have warned you during the install. Remove the duplicated packages.